Barotrauma/BarotraumaClient/ClientSource/Sounds/SoundPlayer.cs

```diff
@@ -26,6 +26,7 @@ static class SoundPlayer
         private static BackgroundMusic previousDefaultMusic;
 
         private static float updateMusicTimer;
+        private static bool increaseUpdateMusicTimer;
```

Comment:
I feel this variable needs a more descriptive name and/or a comment: it's not obvious what it does without reading through the code. `IncreaseUpdateMusicInterval` would seem clearer to me (since what it does is increasing the interval at which the music is updated), and I would instead make this a float - now it's not clear how much it increases the interval, but if it was a float, it'd be clear that it means the amount of time the interval is increased by.

Reply:
I ended up removing this altogether in favour of just adding the current maintrack's value of "MuteIntensityMusic"  to "updateMusicTimer" regardless of whether or not it's monster music (which is what this bool was being used for)